_episodes/03-ncbi-sra.md

```diff
@@ -43,42 +43,49 @@ following 13 sub-projects:"**.
 
-2. Under a section marked "Select." You will see there are 312 runs, 109.43 Gb data, and 168.81 Gbases of data Click on "Metadata". 
+2. At the top of the page is an option to switch back to the “old Run Selector”. We will discuss that later.
+![ncbi-old-runtable](../fig/03_ncbi_new_top.png)
 
-You should now have a file called `SraRunTable.txt`
+3. Notice on this page there are three sections. “Common Fields” “Select”, and “Found 312 Items”. Within “Found 312 Items”, click on the first Run Number (Column “run” Row “1”). 
+![ncbi-new-tables2.png](../fig/03_ncbi_new_tables2.png)
 
-## (Reverting to the old Run Selector) Download the Lenski SRA data from the SRA Run Selector Table
+4. This will take you to a page that is a run browser. Take a few minutes to examine some of the descriptions on the page.
+![ncbi-run-browser.png](../fig/03_ncbi_new_run_browser.png)
 
-1. Access the Tenaillon dataset from the provided link: [https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP064605](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP064605). Click on "Revert to the old Run Selector" at the top of the page. 
-You will be presented with a page for the overall SRA accession SRP064605 - this is a collection of all the experimental data.
+5. Use the browser’s back button to go back to the 'previous page'. As shown in the figure below, the second section of the page (“Select”) has the **Total** row showing you the current number of “Runs”, “Bytes”, and “Bases” in the dataset to date. On 2020-04-27 there were 312 runs, 109.58 Gb data, and 177.17 Gbases of data. 
+![ncbi-new-metadata.png](../fig/03_ncbi_new_metadata.png)
 
-2. Click on the Run Number of the first entry in the bottom table on the page (third table down). This will take you to a page that is a run browser. Take a few minutes to examine some of the descriptions on the page.
+6. Click on the “Metadata” button to download the data for this lesson. The filename is “SraRunTable.txt” and save it on your computer Desktop.
 
-3. Go back to the ['previous page'](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP064605). At the top of the page and in the **Total** row you will see there are 312 runs, 109.43 Gb data, and 168.81 Gbases of data. Click the 'RunInfo Table' button and save the file locally.
+> ## Downloading the Lenski SRA data from the SRA Run Selector Table using the old Run Selector
+> 
+> Go to the [“old Run Selector” instructions](../old-ncbi/index.html) page and return here after downloading your `SraRunTable.txt` file.
+> 
+> We include the “old Run Selector” page because the webpages ***and*** downloaded file `SraRunTable.txt` are slightly different. 
+{: .callout}
```

Comment:
I've converted this into a `callout` so it's rendered in yellow to warn the users.